### PR TITLE
[Outlook] (IdentityAPI) Conditions of using 1.3

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**
Fixes https://github.com/OfficeDev/office-js-docs-pr/issues/2389.

### CODE REVIEW

**Comment by ElizabethSamuel-MSFT** [comment]
Active incident re: build failure: https://portal.microsofticm.com/imp/v3/incidents/details/223868990/home

File: docs/reference/requirement-sets/outlook-api-requirement-sets.md [hunk 1]
```diff
@@ -83,28 +83,30 @@ Add-ins are supported in Outlook on the following platforms.
...
 >
-> <sup>2</sup> Support for 1.3 in Outlook 2013 was added as part of the [December 8, 2015, update for Outlook 2013 (KB3114349)](https://support.microsoft.com/kb/3114349). Support for 1.4 in Outlook 2013 was added as part of the [September 13, 2016, update for Outlook 2013 (KB3118280)](https://support.microsoft.com/help/3118280). Support for 1.4 in Outlook 2016 (one-time purchase) was added as part of the [July 3, 2018, update for Office 2016 (KB4022223)](https://support.microsoft.com/help/4022223).
+> <sup>2</sup> To reference Identity API requirement set 1.3 in your add-in code, you must call `isSetSupported('IdentityAPI', '1.3')`. To directly use an API introduced in that requirement set, you can check that it's not `undefined`. See [Using APIs from later requirement sets](#using-apis-from-later-requirement-sets) for examples.
```
Comment (davidchesnut) [inline docs/reference/requirement-sets/outlook-api-requirement-sets.md]:
```suggestion
> <sup>2</sup> You cannot check that the Identity API requirement set version 1.3 is available by using the `<Requirements>` section in the manifest file. Instead call `isSetSupported('IdentityAPI', '1.3')` at runtime. Or check if the API is `undefined` as described in [Using APIs from later requirement sets](#using-apis-from-later-requirement-sets).
```

**Review by davidchesnut — CHANGES_REQUESTED** [review]
I had a suggested rewrite. Hope it makes sense. Thanks!

**Review by davidchesnut — CHANGES_REQUESTED** [review]
Suggested rewrite if you want to accept/reject. Thanks!

File: docs/overview/office-add-in-availability.md [hunk 2]
```diff
@@ -593,7 +596,10 @@ To work as expected, your Office Add-in might depend on a specific Office applic
   </tr>
 </table>
 
-*&ast; - Added with post-release updates.*
+> [!NOTE]
+> <sup>1</sup> To reference Identity API requirement set 1.3 in your add-in code, you must check if it's supported by first calling `isSetSupported('IdentityAPI', '1.3')`; declaring it in your add-in's manifest isn't supported. To directly use an API introduced in that requirement set, you can check that it's not `undefined`. See [Using APIs from later requirement sets](../reference/requirement-sets/outlook-api-requirement-sets.md#using-apis-from-later-requirement-sets) for example checks.
```
[the remark this answers is not in the document]
Comment (ElizabethSamuel-MSFT) [reply docs/overview/office-add-in-availability.md:600]: Tweaked based on our offline discussion. Thx for your input!

**Pull request merged**